assets/js/all.js

```diff
@@ -269,111 +269,34 @@ var Activiti = Activiti || {};
 
 (function ($) {
   $(function () {
-    var modalsProcessed = {};
```

Comment:
@jsantos-alfresco just FYI, this was a generic modal handler than then got heavily customised for the Marketo specific case, but in its simplest usage was also powering the modal popup on the team listing page, which has now gone away.